### DISCUSSION TITLE
Add radio logos in radio tab

### DISCUSSION THREAD

**Discussion by dorel14** [navidrome/navidrome#2772] [Ideas]
It would be great if we can set the radio picture or logo in radio tab so that whn i import in synfonium i d'on't have to put them manually 

preferable , set an image url

**Comment by byjove01** [comment]
I support this suggestion! It really feels lacking!

**Comment by deluan** [comment]
Symfonium uses the Subsonic API to talk to Navidrome, and the API does not provide support for Radio logos. So this is not something that will be coming anytime soon.

**Comment by karldonteljames** [comment]
This would have been really helpful. :(
Just have to add them to the clients manually.
